File: src/C3.Catalogue/Brands/BrandDraft.vb
Namespace Brands

    Public NotInheritable Class BrandDraft

        Public Sub New(name As String, code As String, notes As String)
            Me.Name = name
            Me.Code = code
            Me.Notes = notes
        End Sub

        Public ReadOnly Property Name As String
        Public ReadOnly Property Code As String
        Public ReadOnly Property Notes As String

    End Class

End Namespace

File: src/C3.Catalogue/Brands/BrandDraft.cs
namespace C3.Catalogue.Brands
{
    public sealed class BrandDraft
    {
        public BrandDraft(string name, string code, string notes)
        {
            Name = name;
            Code = code;
            Notes = notes;
        }

        public string Name { get; }

        public string Code { get; }

        public string Notes { get; }
    }
}
